### ISSUE TITLE
⚡ Build post-reveal page

### ISSUE THREAD

**Issue by Macludde** [Dsek-LTH/web#431]
Here's a todo-list of what needs to be done before it launches:

```[tasklist]
### Needs to be done
- [ ] https://github.com/Dsek-LTH/web/issues/450
- [ ] https://github.com/Dsek-LTH/web/issues/453
- [ ] https://github.com/Dsek-LTH/web/issues/454
- [ ] https://github.com/Dsek-LTH/web/issues/456
- [ ] https://github.com/Dsek-LTH/web/issues/451
- [ ] https://github.com/Dsek-LTH/web/issues/439
- [x] #405
- [x] Move stab and peppers to home screen
- [x] Add messages tab on pre-reveal (or make spoiler-free version of post-reveal)
- [x] Show phaddergroup on ticket (for admins)
```

```[tasklist]
### Nice to have done
- [ ] Add funky design elements to start screen (primarily stab and peppers part)
- [ ] Re-design notifications modal for post-reveal to highlight messages from staben and event stuff
- [ ] Add a notification sent to all subscribers (nollor) when a ticket is about to be released
- [ ] Change design of inventory page (it sucks right now)
- [ ] Show food preference in cart before checkout with ability to edit
- [x] Add copy of settings page to nollning
- [ ] Add mobile browser version of nolla.nu
- [ ] https://github.com/Dsek-LTH/web/issues/436
```
